core/src/main/java/org/mobilitydata/gtfsvalidator/notice/CsvParsingFailedNotice.java

```diff
@@ -52,7 +56,7 @@ public class CsvParsingFailedNotice extends ValidationNotice {
    * @param exception the exception thrown
    */
   public CsvParsingFailedNotice(String filename, TextParsingException exception) {
-    super(SeverityLevel.ERROR);
+    super(ERROR);
```

Comment:
Are we planning to remove the severity field in the future?

Reply:
Yes.  That will be a follow-up refactor.